packages/apps-config/src/endpoints/testingRelayRococo.ts

```diff
@@ -531,6 +531,15 @@ export function createRococo (t: TFunction): EndpointOption {
           Zenlink: 'wss://rococo-parachain.zenlink.pro'
         }
       }
+      { 
```

Comment:
Comma missing on previous line. 

CI will complain about this as well - the entries are expected to be alphabetical.